src/templates/en/2019/base_chapter.html

```diff
@@ -19,20 +19,25 @@
 });
 
 fetch(`https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}.json`)
```

Comment:
Yeah as long as we're creating a chapter-specific script file, let's keep all chapter scripts together.